MangoAPI.Presentation/Startup.cs

```diff
@@ -80,6 +80,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
         app.Map(RoutingConstants.Login, builder => builder.UseSpa(spa => spa.Options.SourcePath = "/wwwroot"));
 
         app.MigrateDatabase();
+        app.InitializeAzureBlob();
```

Comment:
Create Boolean parameters that are in app settings so that if true then migrate database or blob, there must be two flags for each separately